jgiven-junit5/src/test/java/com/tngtech/jgiven/junit5/test/concurrency/JUnit5InheritedParallelizationTest.java

```diff
@@ -0,0 +1,35 @@
+package com.tngtech.jgiven.junit5.test.concurrency;
+
+import com.tngtech.jgiven.junit5.ScenarioTest;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@Execution(ExecutionMode.CONCURRENT)
+class JUnit5InheritedParallelizationTest
+    extends ScenarioTest<Stages.ParallelGivenStage, Stages.ParallelWhenStage, Stages.ParallelThenStage> {
+
+    public static IntStream iterationProvider() {
+        return IntStream.range(0, 100);
+    }
+
+    @ParameterizedTest
+    @MethodSource("iterationProvider")
+    void firstTest(int i) {
+        given().a_thread_local_scenario_state();
+        when().the_state_on_this_thread_is_set_to("I am the greatest" + i);
```

Comment:
For convenience, probably there should be some additional space after greatest. The missing space somehow nudges me :D